BrowserKit/Sources/WebEngine/WKWebview/WKEngineWebView.swift

```diff
@@ -88,6 +98,11 @@ extension WKEngineWebView {
             }
         }
     }
+
+    func replaceLocation(with url: URL) {
+        let safeUrl = url.absoluteString.replacingOccurrences(of: "'", with: WKEngineInfo.apostropheEncoded)
```

Comment:
We should probably just use the built-in foundation utilities for this, something like:

```suggestion
        let charactersToReplace = CharacterSet(charactersIn: "'")
        let safeUrl = url.absoluteString.addingPercentEncoding(withAllowedCharacters: charactersToReplace.inverted)
```